lib/dart/lib/src/console/t_web_socket.dart

```diff
@@ -3,14 +3,14 @@
 /// distributed with this work for additional information
 /// regarding copyright ownership. The ASF licenses this file
 /// to you under the Apache License, Version 2.0 (the
-/// "License"); you may not use this file except in compliance
+/// 'License'); you may not use this file except in compliance
```

Comment:
Not sure why we need to modify the license header **contents**? 
There are more places I will not comment every single one.